docs/DeveloperGuide.md

```diff
@@ -261,13 +261,16 @@ The main Vue object (`window.app`) is responsible for the loading of the report
 - and the tabbed interface
 
 Summary view act as the main report which shows the various calculations. </br>
-Tabbed interface is responsible for loading various modules such as authorship to display additional information.
+Tabbed interface is responsible for loading various modules such as authorship and zoom to display additional information.
 
 ### Javascript Files
 - [**main.js**](../frontend/src/static/js/main.js) - main controller that pushes content into different modules
 - [**api.js**](../frontend/src/static/js/api.js)- loading and parsing of the report content
-- [**v_summary.js**](../frontend/src/static/js/v_summary.js) - module that supports the ramp chart view
-- [**v_authorship.js**](../frontend/src/static/js/v_authorship.js) - module that supports the authorship view
+- [**v_summary.js**](../frontend/src/static/js/v_summary.js) - module that supports the summary view
+- [**v_authorship.js**](../frontend/src/static/js/v_authorship.js) - module that supports the authorship tab view
+- [**v_zoom.js**](../frontend/src/static/js/v_zoom.js) - module that supports the zoom tab view
+- [**v_ramp.js**](../frontend/src/static/js/v_ramp.js) - module that supports the ramp chart view
+- [**v_segment.js**](../frontend/src/static/js/v_segment.js) - module that supports the code segment view
```

Comment:
A little change here, can I have these jump to the each sub sections and the subsection header e.g. The (main.js) in ### Main (main.js) directs user to the file?